widget/richtext.go

```diff
@@ -359,6 +359,7 @@ func (t *RichText) updateRowBounds() {
 				continue
 			}
 			textSeg := seg.(*TextSegment)
+			textSeg.Text = strings.ReplaceAll(textSeg.Text, "\r", "") // remove "\r" characters that otherwise render as space
```

Comment:
My understanding is that this comes as a result of loading files on (for example) windows right? So `SetText` may be a better place to do this (just a 1 time cost).

Reply:
That's a good point. I personally haven't experienced this issue, but you're right, a \r would probably only happen initially. I'll change that.